docs/src/components/PropTable.js

```diff
@@ -201,7 +200,11 @@ export default function PropTable({
                             )}
                           </Td>
                           <Td colspan={1} color="gray">
-                            {Array.isArray(description) ? Description(description) : description}
+                            {Array.isArray(description) ? (
+                              Description(description)
```

Comment:
ONe all docs migrated, Arrays wont be needed right?

Reply:
yup! we can eventually remove that